lib/x265.py

```diff
@@ -63,3 +64,11 @@ def DecodeCommandLine(self, videofile, encodedfile, yuvfile):
                                       encodedfile,
                                       yuvfile)
     return commandline
+
+  def EncoderVersion(self):
+    version_output = subprocess.check_output([encoder.Tool('x265'),
+                                              '--version'],
+                                              stderr=subprocess.STDOUT)
+    # The version is the first line of output.
+    version = version_output.split('\n')[0]
```

Comment:
Fortunately split will return a non-empty list even if version_output is the empty string, otherwise I would have suggested more robust checking here.